packages/components/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "baseUrl": ".",
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
Main reason for this change is that this `postcss-preset-env` update actually changed up some dist structure where it only uses `exports` in their `package.json` so TS was complaining about our usage of it. It only resolves if we use a `moduleResolution` of either `bundler`, `nodenext` or `node16`.

Given `node` is an alias for `node10` and is essentially just based around CJS, it looked like `bundler` would be the most suited way forward. If I understood correctly, this should mean that if our compiled code can be ESM, it will be (and perhaps can remove the errors consumers have been seeing about importing our deps incorrectly?).
https://www.typescriptlang.org/docs/handbook/modules/reference.html#bundler

I didn't use `nodenext` or `node16` because it actually requires any relative paths to be imported with a `.js` extension, which would cause different linting errors.

Alt, we only use `bundler` in the root of the repo so it's all local only, and revert the KAIO one to `node` so that the dist content doesn't change (not really sure if it even changes now - haven't checked). The main problem would be the discrepancy would possibly mean we aren't seeing a correct reflection of errors we might encounter in one over the other.